Sources/PointFree/SiteMiddleware.swift

```diff
@@ -106,9 +106,13 @@ private func render(conn: Conn<StatusLineOpen, T3<Database.Subscription?, Databa
         |> episodeResponse
 
     case let .expressUnsubscribe(userId, newsletter):
-      return conn.map(const(user .*. userId .*. newsletter .*. unit))
+      return conn.map(const(userId .*. newsletter .*. unit))
```

Comment:
current user isnt needed because "express unsubscribe" should work regardless of who is logged in/out